### PR TITLE
general functionality of edit page, still needs work

### PR DESCRIPTION
**Pull request by karlscheib**
right now, I can only get one image to display at a time

### CODE REVIEW

**Comment by kudlakk** [comment]
nobody merge this yet, karl said he's still adding to it and just wanted our input

File: src/routes/FFC_Edit.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return<>
+    <form onSubmit={(e) => { handleOnSubmit(e) }}>
+      <label >
+        Deck Name
+      </label><br />
+      <input type="text" value={deckName} required onChange={(e)=> setDeckName(e.target.value)} /><br />
+      <div>
+        {selectedImage && (
+          <div>
+          <img alt="not fount" width={"250px"} src={URL.createObjectURL(selectedImage)} /> <br />
```
Comment (kudlakk) [inline src/routes/FFC_Edit.js]: fount

**Review by kudlakk — COMMENTED** [review]
everything seems pretty fine so far to me

**Comment by karlscheib** [comment]
The most recent push is for the deck select page. Right now has some temporary example decks to show it working. Still needs to be connected to the backend and connected to the other pages once those are complete

**Comment by kudlakk** [comment]
don't forget to resolve merge conflicts with main.

`git checkout main`
`git pull`
`git checkout flash-cards`
`git merge main`

then resolve conflicts and push to this branch

**Comment by kudlakk** [comment]
will re-open when ffc branch is done

**Pull request closed without merging**